server/templates/base_page.html

```diff
@@ -1,6 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <script async>
```

Comment:
Yes, we shouldn't be adding the `async` to the inline script tag. What race condition do you mean? When the browser hits this `<script>` tag it will execute its contents immediately before going on to parse the rest of the HTML file (in theory).